utils/scripts/reportportal/rp_uploader.py

```diff
@@ -0,0 +1,102 @@
+import os
+import argparse
+import sys
+import json
+
+dir_path = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(dir_path+"/../")
+
+from util import execute_command
+from logger import log
+
+"""
+Class for Report portal operations
+"""
+
+
+class reportPortalOperations():
+    def __init__(self, args={}):
+        # Initialize instance variables
+        self.config_file = args.get("config_file")
+        self.payload_dir = args.get("payload_dir")
+        self.service_url = args.get("service_url")
+        self.output_file = args.get("output_file")
+        self.log_path = args.get("log_path")
+
+    def write_output_file(self, file_content):
+        """
+        Write content to output file.
+        Args:
+            file_content (str): Content to write to output file.
+        """
+
+        output_file = os.path.abspath(os.path.expanduser(self.output_file))
+        try:
+            if not os.path.exists(os.path.dirname(output_file)):
+                os.makedirs(os.path.dirname(output_file))
+            with open(output_file, "a") as f:
+                log.info("Writing content to %s", output_file)
+                f.write(f"{file_content}\n")
+        except Exception as e:
+            log.error("Unable to write output to file due to Exception: %s", e)
+
+    def upload_result(self):
+        """Uploads test results to report portal"""
+
+        cmd = ("rp_preproc -c {} -d {} "
+               "--service {} -l {}".format(self.config_file,
+               self.payload_dir, self.service_url, self.log_path))
+        log.info("CMD: {}".format(cmd))
+        rp_output = execute_command(cmd)
+        rp_output_json = json.loads(rp_output)
+        self.write_output_file(json.dumps(rp_output_json))
+
+
+if __name__ == "__main__":
+
+        #Instance for reportPortalOperations Class
+        rp_obj = reportPortalOperations()
+
+        """Parse CLI arguments"""
+
+        parser = argparse.ArgumentParser(
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            description='Script to upload results to report portal')
+
+        subparsers = parser.add_subparsers(title='Available sub commands',
+                                           help='sub-command help')
+
+        #Argument parsers for create_cluster
```

Comment:
Comment should be changed to something like  #Argument parsers for upload_rp_result

Reply:
Comments Incorporated